src/test/java/com/artipie/asto/RxFileTest.java

```diff
@@ -66,18 +66,21 @@ public void rxFileSaveWorks() throws IOException {
         final Vertx vertx = Vertx.vertx();
         final String hello = "hello-world!!!";
         final Path temp = Files.createTempFile(hello, "saved.txt");
-        temp.toFile().delete();
-        new RxFile(temp, vertx.fileSystem())
-            .save(
-                Flowable.fromArray(new ByteArray(hello.getBytes()).boxedBytes()).map(
-                    aByte -> {
-                        final byte[] bytes = new byte[1];
-                        bytes[0] = aByte;
-                        return ByteBuffer.wrap(bytes);
-                    }
-                )
-            ).blockingAwait();
-        MatcherAssert.assertThat(new String(Files.readAllBytes(temp)), Matchers.equalTo(hello));
+        final int iterations = 100;
+        for (int idx = 0; idx < iterations; idx += 1) {
```

Comment:
@Sammers21 maybe you can use `@RepeatedTest` [Junit annotation](https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/RepeatedTest.html) instead of for loop here?